libraries/AP_Mount/AP_Mount_SoloGimbal.cpp

```diff
@@ -115,15 +115,31 @@ void AP_Mount_SoloGimbal::set_mode(enum MAV_MOUNT_MODE mode)
     _mode = mode;
 }
 
-// send_mount_status - called to allow mounts to send their status to GCS using the MOUNT_STATUS message
-void AP_Mount_SoloGimbal::send_mount_status(mavlink_channel_t chan)
+// send a GIMBAL_DEVICE_ATTITUDE_STATUS message to GCS
+void AP_Mount_SoloGimbal::send_gimbal_device_attitude_status(mavlink_channel_t chan)
```

Comment:
Make this non-virtual and create `virtual bool AP_Mount_Backend::get_attitude_quaterniion(Quatnerion &quat)`

Reply:
done, txs